src/main/kotlin/com/terraformation/backend/accelerator/variables/StableId.kt

```diff
@@ -1,3 +1,8 @@
 package com.terraformation.backend.accelerator.variables
 
-data class StableId(val value: String)
+import com.terraformation.backend.db.docprod.VariableType
+
+data class StableId(
+    val value: String,
+    val variableType: VariableType,
+)
```

Comment:
I see what this change is for on the test side, but this change means that `StableId` is no longer a type-safe wrapper for an underlying string value which was the reason we added it.

The semantics change a bit: `StableId("123", VariableType.Text)` and `StableId("123", VariableType.Number)` are now both valid values but they aren't equal to each other despite having the same ID value. This could be more than a theoretical concern in contexts like importing the all-variables CSV, where the type of a variable can change and we compare stable IDs to match the old and new versions.

I think a better approach here, since `variableType` is never used anywhere in non-test code, would be to add a `Map<StableId, VariableType>` constant in the test code to keep the type information. Then the application code can continue to treat `StableId` as a simple type-safe ID value, which will mean this change needs to touch much less code.